sample.env

```diff
@@ -0,0 +1,2 @@
+USERNAME=
```

Comment:
Remove this file. README instruction is good enough.

Reply:
Done